alvr/dashboard/src/steamvr_launcher/mod.rs

```diff
@@ -129,6 +130,8 @@ impl Launcher {
 
         alvr_server_io::driver_registration(&[alvr_driver_dir], true).ok();
 
+        data_sources::get_local_session_source().session_mut();
```

Comment:
This feels like a hack. But i think the original code was racey anyway. I have plans to pass the settings to the c++ side without going through the session.json file, but this will do for now

Reply:
`session_mut()` is called only when something is changed so it should be created somewhere anyway. maybe it should be triggered by making default `server_version` not equal to current version.

Reply:
I'm reading now your message on Discord, you probably should move the session initialization back to the driver, in place of the path backup mechanism